server/hotrod/src/main/java/org/infinispan/server/hotrod/DefaultSaslServerFactoryFactory.java

```diff
@@ -0,0 +1,39 @@
+package org.infinispan.server.hotrod;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.function.BiFunction;
+
+import javax.security.sasl.SaslServerFactory;
+
+import org.infinispan.commons.util.SaslUtils;
+
+/**
+ * Meta-factory for available {@link SaslServerFactory}
+ *
+ * @author Tristan Tarrant &lt;tristan@infinispan.org&gt;
+ * @since 10.0
+ **/
+public class DefaultSaslServerFactoryFactory implements BiFunction<String, Map<String, ?>, SaslServerFactory> {
```

Comment:
Factory for a Factory? Can this not just be `DefaultSaslServerFactory`?

Reply:
The class was unused. it's gone